client/packages/common/src/intl/locales/en/common.json

```diff
@@ -743,11 +744,13 @@
   "label.collapse-all": "Collapse all",
   "label.color": "Colour",
   "label.comment": "Comment",
+  "label.confirmation-date": "Confirmation date",
```

Comment:
Do we already have a confirmed date? Won't this show in the progress bar like this?
@mark-prins  do we want this kind of progress bar added? Can't remember if it's in the mockup?

<img width="688" height="481" alt="image" src="https://github.com/user-attachments/assets/98282769-0e4a-4fee-85fd-21473e3ed01a" />

Reply:
Will bring this discussion over to this PR: https://github.com/msupply-foundation/open-msupply/pull/8690. As I've implemented it there and have left questions relating to this as well 😄